skyve-war/src/main/java/modules/admin/User/UserService.java

```diff
@@ -43,6 +45,11 @@
  */
 @Default
 public class UserService {
+	@Inject
+	Persistence pers;
+
+	@Inject
+	GroupService groupService;
```

Comment:
I think the Skyve injected classes are already safe from serialisation, but we should declare these as `private transient` just in case, and it reinforces the pattern for other developers to copy when injecting classes.

GroupService should definitely be `private transient`.